fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -4042,7 +4042,7 @@ public void modifyTableColocate(Database db, OlapTable table, String assignedGro
                 }
             } else {
                 // set to an already exist colocate group, check if this table can be added to this group.
-                groupSchema.checkColocateSchema(table);
```

Comment:
why not check all partitions in this case? 
And please add comment here , or other people will not know the reason and may delete it during refactoring.

Reply:
We add a table with many partitions to the colocate group, and we need to check whether all partitions comply with the colocate group specification

Reply:
> why not check all partitions in this case? And please add comment here , or other people will not know the reason and may delete it during refactoring.

We add a table with many partitions to the colocate group, and we need to check whether all partitions comply with the colocate group specification